src/cmdstan-guide/installation.Rmd

```diff
@@ -375,15 +375,66 @@ and in working order, run the following series of commands:
 The sample data in file `bernoulli.json.data` specifies 2 out of 10 successes, therefore
 the range `mean(theta)`$\pm$`sd(theta)` should include 0.2.
 
+## Troubleshooting the installation
+
 Updates to CmdStan or changes in compiler options may result in errors
 when trying to compile a Stan program.  In some cases, these can be resolved
-by removing the existing CmdStan build and recompiling.
-The Makefile target `clean-all` should be run before rebuilding CmdStan:
+by removing the existing CmdStan binaries and recompiling.
+
+To do this, you must run the makefile commands from the `<cmdstan-home>` directory:
+
 ```
-> make clean-all
+> cd <cmdstan-home>
+> make clean-all    # mingw32-make if in Windows
 > make build
 ```
 
+### Common problems
+
+This section contains solutions to problems reported on https://discourse.mc-stan.org
+
+**Compiler error message about PCH file**
+
+To speed up compilation, the Stan makefile pre-compiles parts of the core Stan library.
+If these pre-compiled files are out of sync with the compiled model, the compiler will complain, e.g.:
+
+```
+error: PCH file uses an older PCH format that is no longer supported
+```
+
+In this case, clean and rebuild CmdStan, as shown in the previous section.
+
+
+**Windows: ‘mingw32-make’ is not recognised**
+
+If the C++ toolchain has been installed but no properly registered,
+the call to `mingw32-make` will fail with the error message:
+
+```
+‘mingw32-make’ is not recognised as an internal or external command
+```
+
+The solution is to call utility `pacman` as follows:
+
+```
+pacman -Sy mingw-w64-x86_64-make
+```
+
+See: https://discourse.mc-stan.org/t/cmdstan-installation-on-windows/11287/6
```

Comment:
This is only true if you use the RTools 4.0 toolchain. 

Rtools35: 

- if mingw32-make, g++ or cut is not found, you need to add `C:\Rtools\mingw_64\bin` and `C:\Rtools\bin` to PATH

RTools40: 

- if mingw32-make, g++ or cut is not found you need to add  `C:\rtools40\mingw64\bin`  and `C:\rtools40\usr\bin`  and then run the pacman command. 

And that is it.

Reply:
thanks!